content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -6,7 +6,7 @@ weight: 110
 
 {{% capture overview %}}
 
-This page shows how to configure liveness and readiness probes for Containers.
+This page shows how to configure health probes for Containers.
```

Comment:
Does the page title need to change as well?

Reply:
I would say no, because the stars of the show are still liveness and readiness... we only add a small use case for startup to improve liveness.

Reply:
I will then revert the overview sentence, thanks for pointing out the inconsistency!